config.schema.yaml

```diff
@@ -0,0 +1,67 @@
+---
+  url:
+    description: "URL to Docker socket"
```

Comment:
Minor thing - perhaps it would be better to say something along the lines of "Path to the socket file or URI for the docker deamon".

Since in theory it doesn't necessary need to be a socket.